app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -308,6 +323,15 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         }
     }
 
+    @VisibleForTesting(otherwise = PROTECTED)
+    internal open fun getIntentStartSource(intent: SafeIntent): Event.OpenedAppAllStart.Source? {
+        return when {
+            intent.isLauncherIntent -> Event.OpenedAppAllStart.Source.APP_ICON
+            intent.action == Intent.ACTION_VIEW -> Event.OpenedAppAllStart.Source.LINK
+            else -> null
```

Comment:
Will update it!

Reply:
@mcomella  if we record unknown, should we at least store the value of the "UNKNOWN" so that we have some insight on what those different starts were in order to adapt the code afterwards?